_posts/2020-01-03-municipal-boundaries-final-update-for-2019.md

```diff
@@ -0,0 +1,44 @@
+---
+title: 'Municipal Boundaries Final update for 2019'
+author:
+  display_name: Mike Heagin
+  email: mheagin@utah.gov
+date: 2020-01-03 17:19:47
+categories:
+  - SGID Blog
+tags:
+  - municipality
+  - boundaries
+  - annexations
+---
+
+Once again we come to the end of the year, nothing we can do to prevent it, time move on. And so does progress, as the folks in Utah County know, they are growing all the time. They lead again in the number of annexation this year with 28, not too far behind is Cache County with 20. Both were far ahead of the other counties out there, almost three times the number of annexations. Again this year we are welcoming a new city to the state, welcome  to the Town of Brighton. Many are familiar with the area, now they are an incorporated city as well as a great place to ski and tour. Thanks to all the hard working Recorders and staff out there in the counties and towns. With your help the Lt. Governor’s Office and AGRC processed 104, annexation, adjustments, disconnections and incorporations. Special thanks to Emery and Weber Counties for helping get the state data up to date, by sending us a couple missing annexations. As always feel free to check out all the changes and make sure your boundaries are where you think they should be, view them here: [Utah Municipality Modifications](https://www.arcgis.com/home/webmap/viewer.html?webmap=c5ab7e0fcd514f1a9db6b8dad55bba63).
```

Comment:
```suggestion
Once again we've come to the beginning of a new year. There's nothing we can do to prevent it; time moves on and progress is made, as the folks in Utah County know. Utah County is growing all the time. As in 2018, they led again in the number of annexations in 2019 with 28. Not too far behind was Cache County with 20. Both were far ahead of the other counties --- almost three times the number of annexations. Also this year we are welcoming another new city to the state: the Town of Brighton. Many are familiar with the area, and now the Town of Brighton is an incorporated city as well as a great place to ski and tour. We would like to thank all the hardworking recorders and staff out there in the counties and towns. With your help, the Lieutenant Governor’s Office and AGRC processed 104 annexations, adjustments, disconnections, and incorporations in 2019. We would also like to extend a special thank-you to Emery County and Weber County for helping get the state data up to date by sending us a couple of missing annexations. 

As always, feel free to check out all the changes and make sure your boundaries are where you think they should be. You can view them here: [Utah Municipality Modifications](https://www.arcgis.com/home/webmap/viewer.html?webmap=c5ab7e0fcd514f1a9db6b8dad55bba63).
```